tests/functional/generate_pages.py

```diff
@@ -69,6 +69,13 @@
 
 """
 
+RELATIONSHIP_RADIO_CLICKER = r"""  click{optionName}() {
+    browser.element('[id="{optionId}"]').click().pause(300)
```

Comment:
Agreed but the overhead of listening for `transitionend` events isn't worth it—it's not very reliable and requires a library to solve the cross-browser issues. Also, if we removed the transition or changed it to an `animation` (different CSS property) we'd also have to update the JS.